e2e/utils/element-utils.ts

```diff
@@ -7,24 +7,24 @@ import {ElementHandle, JSHandle, Page} from 'puppeteer';
  * @param {string} attribute Attribute name.
  */
 export const getAttrValue = async (page: Page, element: ElementHandle, attribute: string): Promise<string> => {
-  return page.evaluate((link, attr) => link.getAttribute(attr), element, attribute);
+  return page.evaluate((link, attr) => {return link.getAttribute(attr)}, element, attribute);
 }
 
 /**
  * Gets the property value.
  * @param {ElementHandle} element Element.
  * @param {string} property Property name.
  */
-export const getPropValue = async (element: ElementHandle, property: string): Promise<string> => {
+export async function getPropValue<T> (element: ElementHandle, property: string): Promise<T> {
   const value = await element.getProperty(property).then((prop) => prop.jsonValue());
-  return value.toString().trim();
+  return value as T;
 }
 
 /*
  * Wait until the existing element changed in DOM.
  * @param {Page} page Instance of Puppeteer page object.
  * @param {ElementHandle} element Element.
  */
-export async function waitUntilChanged(page: Page, element: ElementHandle): Promise<JSHandle> {
+export const waitUntilChanged = async (page: Page, element: ElementHandle): Promise<JSHandle> => {
```

Comment:
What is the purpose of this change? This seems like the opposite syntax compared to the method above.

Reply:
forgot to change it back. I was experimenting.